docs/software/dns_servers.md

```diff
@@ -215,4 +216,82 @@ DietPi Blog: [`Pi-Hole & Unbound: How to have ad-free & safer internet in just f
 
 Blog entry with YouTube video (German language): [`Unbound Installation für PiHole unter DietPi`](https://blog.login.gmbh/unbound-installation-fuer-pihole-unter-dietpi/)
 
+## AdGuard Home
+
+AdGuard Home is a DNS sinkhole with web interface that will block ads for any device on your network.
+
+![AdGuard Home web interface screenshot](../assets/images/dietpi-software-dnsserver-adguardhome.png){: width="500" height="410" loading="lazy"}
+
+=== "Access the web interface"
+
+    The web interface is accessible via port **8083**:
+
+    - URL = `http://<your.IP>:8083`
+    - User = `admin`
+    - Password = `<yourGlobalSoftwarePassword>` (default: `dietpi`)
+
+=== "Configuration"
+
+    The configuration contains setting devices (e.g. router) to use AdGuard Home for DNS resolution.
+
+    <font size="+2">Option 1 - Setup single devices to use the AdGuard Home DNS server</font>
+
+    Simply change your DNS settings to use the IP address of your AdGuard Home device. This will need to be done for each device that you want AdGuard Home to work with.
+
+    Example:
+
+    - My AdGuard Home device has the IP address of 192.168.0.100
+    - On my PC, I would set the DNS address to 192.168.0.100
+    - Tutorial [The Ultimate Guide to Changing Your DNS settings](https://www.howtogeek.com/167533/the-ultimate-guide-to-changing-your-dns-server/).
+
+    <font size="+2">Option 2 - Setup your router to use the AdGuard Home DNS server</font>
+
+    This method will automatically point every device (that uses DHCP) on your network to AdGuard Home.
+    On your routers control panel web page, you will need to find a option called "DNS server". This should be located under DHCP settings.
+
+    Simply enter the IP address of your AdGuard Home device under "DNS server":
+
+    ![DietPi DNS server software router setup](../assets/images/dietpi-software-dnsserver-router-setup.png){: width="400" height="240" loading="lazy"}
+
+    On your AdGuard Home device, you will need to set a different DNS server.  
+    Depending on your router configuration, if you don't do this step, the AdGuard Home device may not be able to access the internet. It's highly recommended to have the device running AdGuard Home, pointing to a DNS server outside your network.
+
+    - Run the following command: `dietpi-config 8 1`
+    - Select: *Ethernet*
+    - If you are running in DHCP mode, select *Change Mode*, then select: *Copy Current address to Static*
+    - Select *Static DNS* from the list, then choose a DNS server, or manually enter a custom entry.
+    - Once completed, select *Apply* to save the changes.
+
+=== "Updating AdGuard Home"
+
+    To update AdGuard Home, simply reinstall it:
+
+    ```sh
+    dietpi-software reinstall 126
```

Comment:
But it has an internal updater as well, which should be preferred as long as web UI (didn't check CLI) works.

Reply:
Nope it will just show the new version and link you to GitHub. At least this was the behaviour on my test.

Reply:
Ah, good to know. I found a post in their forums which seems to imply that an updater exist, but probably I misinterpreted it or it did but does not anymore.

Reply:
I will run a rest again